src/sql/Expr.h

```diff
@@ -188,8 +193,18 @@ struct Expr {
 
   static Expr* makeDateLiteral(char* val);
 
+  static Expr* makeTimestampLiteral(char* val);
+
+  static Expr* makeTimeLiteral(char* val);
+
   static Expr* makeIntervalLiteral(int64_t duration, DatetimeField unit);
 
+  static Expr* makeCurrentTimestamp();
+
+  static Expr* makeCurrentDate();
+
+  static Expr* makeCurrentTime();
```

Comment:
Please see my previous comment. The "current" versions of these functions are not required when we always pass a `char*` for the name (probably `nullptr`) and the flag indicating whether it's the "current" version.